CHANGES.md

```diff
@@ -1,9 +1,11 @@
 1.2.0
 =====
 
+* [#170](https://github.com/serokell/universum/pull/170):
+  Remove `ElementConstraint` from the `Container` class.
```

Comment:
1.2.0 is already on Hackage, so this changelog entry is misplaced. It should probably go to 1.3.0. @ChShersh?

Reply:
@gromakovsky Yes, this should go to `1.3.0`. I don't want to bother @effectfully with maintaining changelog properly since it involves knowing specific version policy for the project, etc. I can move this line to `1.3.0` w/o problems but I'm just to lazy to write everything by my own...